app/views/unattended/provisioning_templates/script/windows_default_script.erb

```diff
@@ -88,11 +88,6 @@ copy x:\windows\system32\sdelete.exe C:\Windows\
 IF not exist C:\Windows\Setup\Scripts (md C:\Windows\Setup\Scripts)
 echo call C:\deploy\foreman-finish.bat ^> c:\foreman.log 2^>^&1 > C:\Windows\Setup\Scripts\SetupComplete.cmd
 
-<% if foreman_url('user_data') %>
-echo Downloading user data
-%WGET% <%= foreman_url('user_data') %> -O c:\deploy\joinDomain.ps1
-<% end -%>
-
```

Comment:
Hey, thanks.

Can you keep this block and execution of its content? Just rename it to something generic like `user_data.ps1` or whatever but it is useful to be able to inject arbitrary commands via userdata template.

Other than that, I don't have a Windows environment. Can you make sure the last version is tested please?

Reply:
Good Idea, with this users can inject their own powershell scipts

Reply:
in our enviroment windows installation and domain join with secure and unsecure mode works great